system/forms/formbuilder/item-types/formfield.xml

```diff
@@ -18,6 +18,7 @@ it will be merged with this form definition.
 			<field name="layout"    control="formbuilderFieldLayoutPicker" required="true"  sortorder="30" />
 			<field name="mandatory" control="yesnoswitch"                  required="false" sortorder="40" />
 			<field name="help"      control="textarea"                     required="false" sortorder="50" />
+			<field name="export"    control="yesnoswitch"                                   sortorder="60" />
```

Comment:
default="true" was removed due to the front end will always show "true" even though it was set to false. This will happen when user navigate out from the config page and reloading. Removing the default will solve the problem.